mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -1466,7 +1466,8 @@ static LogicalResult reductionPreconditions(LinalgOp op) {
 
 static LogicalResult vectorizeDynamicLinalgOpPrecondition(linalg::LinalgOp op) {
   // TODO: Masking only supports dynamic generic ops for now.
```

Comment:
We could probably remove this TODO and add a comment on that these are the ops for which dynamic shapes vectorization is enabled.

Reply:
Good point, thanks! I also remove linalg.fill op from the list because it is an element-wise op.